### PR TITLE
docs(edits): update pica use cases + edit centauri chain name

### PR DESCRIPTION
**Pull request by JafarAz**
Required for merge:
- [ ] `pr-workflow-check / draft-release-check` is ✅ success
- Other rules GitHub shows you, or can be read in [configuration](../terraform/github.com/branches.tf) 

Makes review faster:
- [x] PR title is my best effort to provide summary of changes and has clear text to be part of release notes 
- [x] I marked PR by `misc` label if it should not be in release notes
- [ ] Linked Zenhub/Github/Slack/etc reference if one exists
- [ ] I was clear on what type of deployment required to release my changes (node, runtime, contract, indexer, on chain operation, frontend, infrastructure) if any in PR title or description
- [x] Added reviewer into `Reviewers`
- [ ] I tagged(`@`) or used other form of notification of one person who I think can handle best review of this PR
- [ ] I have proved that PR has no general regressions of relevant features and processes required to release into production
- [ ] Any dependency updates made, was done according guides from relevant dependency
- Clicking all checkboxes 
- Adding detailed description of changes when it feels appropriate (for example when PR is big)

### CODE REVIEW

File: docs/docs/networks/composable/LAYR-distribution-diagram.png
Comment (0xBrainjar2) [inline docs/docs/networks/composable/LAYR-distribution-diagram.png:1]: series a and seed should be merged to just say token purchasers, but in the description you should delineate what % is vesting when (ie xyz vesting x and xyz vesting y)

File: docs/docs/networks/composable/LAYR-tokenomics.md [hunk 1]
```diff
@@ -45,7 +45,10 @@ The total supply of 100 million LAYR tokens are intended to be distributed and r
...
 
-**Token purchasers**: ~23.7% (23,710,772 LAYR) of the total token supply will be distributed to token purchasers as a reward for their essential early support and strategic backing of the ecosystem. 20% (4,742,154.34) of these tokens will be unlocked at the token generation event (TGE), with linear vesting of the remaining tokens over the subsequent two years.
+**Token purchasers**: 23.71% (23,710,772 LAYR) of the total token supply will be distributed to token purchasers as a reward for their essential early support and strategic backing of the ecosystem. These are split between two rounds: 
+- i) 10.36% Seed Allocation (13,359,572 LAYR) — 20% (4,742,154.34 LAYR) of these tokens will be unlocked at the token generation event (TGE), with linear vesting of the remaining tokens over the following twenty-four months.
```
Comment (0xBrainjar2) [inline docs/docs/networks/composable/LAYR-tokenomics.md:49]: dont delineate this way, see comment above

File: docs/docs/networks/picasso/pica-use-cases.md [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
+And so, PICA was born a natively cross-chain token. Powering multiple chains. Additional value will accrue to the token based on the success of IBC's ongoing cross-pollination efforts. PICA is the native token of [Picasso](../picasso-parachain-overview.md) and [Composable Cosmos](../composable-cosmos.md).
+
+A concerted effort has been made to ensure that the PICA token holds as much utility as possible by incorporating various value accrual methods and governance features. While the PICA token provides rewards for participating within the ecosystem, it is also fundamental for the operation of collators, validators, oracles, and other cross-ecosystem strategies.
+
+## What is $PICA used for?
+A summary of PICA use cases is as follows:
+
+- Collator staking on Picasso 
+- Apollo staking
+- Validator Staking
```
Comment (0xBrainjar2) [inline docs/docs/networks/picasso/pica-use-cases.md:14]: add revenue

File: docs/docs/networks/picasso/pica-use-cases.md [hunk 3]
```diff
@@ -14,36 +24,63 @@ Apollo is a permissionless, MEV-resistant oracle solution. Anyone can run an ora
...
-### Powering our advancement into the Cosmos
-New use cases will continue to be established as we move forward with XCM channel openings and our expansion into the Cosmos.
+### Powering advancements in the Cosmos
+New use cases will continue to be established as we move forward with HRMP channel openings and expansion into and beyond the Cosmos.
 
-## Governance & treasury management
+## Governance
 
-The treasury will be community owned and controlled by PICA token holders.
+Picasso is waging war on centralization with a vision of a seamlessly interoperable, trustless future for DeFi. As such the PICA token will play an important role in helping to realize this vision, as governance will soon be powered by phase 2 of Picasso's [OpenGov](./governance.md).
```
Comment (0xBrainjar2) [inline docs/docs/networks/picasso/pica-use-cases.md:83]: and governance on cosmos chain , please see the docs i edited and left comments for in the rebrand channel

File: docs/docs/networks/picasso/token-transparency.md [hunk 4]
```diff
@@ -106,19 +107,29 @@ Pursuant to [Council motion #19](https://picasso.polkassembly.io/motion/19), the
...
 Token purchasers’ funds have been loaded into a vesting pallet and
 will unlock based on the vesting schedule specified on the PICA token distribution. 
 For more on how the vesting pallet works, 
 please read: [https://github.com/paritytech/substrate/blob/master/frame/vesting/src/lib.rs.](https://github.com/paritytech/substrate/blob/master/frame/vesting/src/lib.rs)
 
+### Composable Cosmos Allocation
+
+The multisig address `centauri12k2pyuylm9t7ugdvz67h9pg4gmmvhn5vmvgw48` holds 1,066,669,217.2 PICA for validator delegations to secure the Composable Cosmos chain. 
+
+### Hyperspace Relayer allocation
```
Comment (0xBrainjar2) [inline docs/docs/networks/picasso/token-transparency.md:130]: add purpose

File: docs/sidebars.js [hunk 5]
```diff
@@ -56,7 +56,7 @@
              "user-guides/claim-rewards-guide",
              "user-guides/how-to-provide-liquidity",
              "user-guides/how-to-trade-pica-on-pablo",
-             "user-guides/centauri-staking",
```
Comment (0xBrainjar2) [inline docs/sidebars.js:59]: guide for adding kepler

File: docs/docs/networks/composable/LAYR-tokenomics.md (hunk 1 above)
Comment (JafarAz) [reply docs/docs/networks/composable/LAYR-tokenomics.md:49]: this aligns with your comment above

File: docs/docs/networks/composable/LAYR-tokenomics.md [hunk 6]
```diff
@@ -45,7 +45,10 @@ The total supply of 100 million LAYR tokens are intended to be distributed and r
 
 **Team**: 25% (25,000,000 LAYR) of the total token supply will be distributed to founders and the core Composable team as a reward as well as incentives for their continued actions in the best interest in the Composable ecosystem. The distribution of these tokens will have a six-month cliff with linear vesting of the remaining tokens over the subsequent two and a half years.
 
-**Token purchasers**: ~23.7% (23,710,772 LAYR) of the total token supply will be distributed to token purchasers as a reward for their essential early support and strategic backing of the ecosystem. 20% (4,742,154.34) of these tokens will be unlocked at the token generation event (TGE), with linear vesting of the remaining tokens over the subsequent two years.
+**Token purchasers**: 23.71% (23,710,772 LAYR) of the total token supply will be distributed to token purchasers as a reward for their essential early support and strategic backing of the ecosystem. These are split between two rounds: 
+- i) 13.36% Seed Allocation (13,359,572 LAYR) — 20% (4,742,154.34 LAYR) of these tokens will be unlocked at TGE, with linear vesting of the remaining tokens over the following twenty-four months.
```
Comment (0xBrainjar2) [inline docs/docs/networks/composable/LAYR-tokenomics.md:49]: just say 13.6 vested with vesting schedule A. And 10.36 vesting swith schedule B no need to differentiate rounds

File: docs/docs/networks/picasso/pica-use-cases.md [hunk 7]
```diff
@@ -14,36 +25,65 @@ Apollo is a permissionless, MEV-resistant oracle solution. Anyone can run an ora
...
 
-### Powering our advancement into the Cosmos
-New use cases will continue to be established as we move forward with XCM channel openings and our expansion into the Cosmos.
+## Governance
 
-## Governance & treasury management
+Picasso is waging war on centralization with a vision of a seamlessly interoperable, trustless future for DeFi. As such the PICA token will play an important role in helping to realize this vision, as governance will soon be powered by phase 2 of Picasso's [OpenGov](./governance.md).
 
-The treasury will be community owned and controlled by PICA token holders.
+As PICA also serves as the native token of Composable Cosmos, it is required for governance of the chain aswell.
```
Comment (0xBrainjar2) [inline docs/docs/networks/picasso/pica-use-cases.md:86]: speak to joe - describe how the treasury is managed currently - like using council, and then migrating to public at some point

File: docs/docs/user-guides/composable-cosmos-staking.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+1. To begin, you need to send PICA from Picasso to Composable Cosmos. Go to https://app.trustless.zone/ and connect both your Polkadot and Cosmos wallets.
+
+![staking-uno](./images-staking-centauri/centauri-stake-1.png)
+
+1. Select Picasso as the source chain and Centauri as the destination chain. Input the desired amount you want to send to Composable Cosmos for staking. A pop-up will appear, prompting you to confirm the specified amount.
+
+![staking-deus](./images-staking-centauri/centauri-stake-2.png)
+
+3. Now head to https://ping.pub/composable and connect your Keplr wallet. Select the 'Delegate' button and the validator you wish to delegate your PICA tokens to. Enter the amount of PICA you wish to stake.
+
```
Comment (0xBrainjar2) [inline docs/docs/user-guides/composable-cosmos-staking.md:14]: add kepler addition guide

File: docs/docs/networks/picasso/pica-use-cases.md (hunk 7 above)
Comment (JafarAz) [reply docs/docs/networks/picasso/pica-use-cases.md:86]: yes I planned to add this once Opengov was fully implemented on Picasso

File: docs/docs/networks/composable/LAYR-tokenomics.md [hunk 9]
```diff
@@ -45,13 +45,13 @@ The total supply of 100 million LAYR tokens are intended to be distributed and r
 
 **Team**: 25% (25,000,000 LAYR) of the total token supply will be distributed to founders and the core Composable team as a reward as well as incentives for their continued actions in the best interest in the Composable ecosystem. The distribution of these tokens will have a six-month cliff with linear vesting of the remaining tokens over the subsequent two and a half years.
 
-**Token purchasers**: ~23.7% (23,710,772 LAYR) of the total token supply will be distributed to token purchasers as a reward for their essential early support and strategic backing of the ecosystem. 20% (4,742,154.34) of these tokens will be unlocked at the token generation event (TGE), with linear vesting of the remaining tokens over the subsequent two years.
+**Token purchasers**: 23.71% of the total token supply (23,710,772 LAYR) will be distributed to early supporters and strategic backers, with 13.36% (13,359,572 LAYR) in the Seed Allocation, where 20% unlocks at TGE, and the rest vests over 24 months, and 10.36% (10,351,000 LAYR) in the Series A Allocation, with a 6-month vesting period followed by linear vesting over 24 months.
```
Comment (0xBrainjar2) [inline docs/docs/networks/composable/LAYR-tokenomics.md:48]: sir for the 4th time lol just say this: 
23.71% of the total token supply (23,710,772 LAYR) will be distributed to early supporters and strategic backers, with 13.36% (13,359,572 LAYR) unlocking 20% at TGE, and the rest vesting over 24 months, and 10.36% (10,351,000 LAYR) with a 6-month vesting period followed by linear vesting over 24 months.

Comment (0xBrainjar2) [reply docs/docs/networks/composable/LAYR-tokenomics.md:48]: 23.71% of the total token supply (23,710,772 LAYR) will be distributed to early supporters and strategic backers. 13.36% (13,359,572 LAYR) of this allocation will  have a 20% unlock at TGE, with the rest vesting over 24 months. 10.36% (10,351,000 LAYR)  of this allocation will be locked for 6-month vesting period followed by linear vesting over 24 months.

**Pull request merged**